packages/react-components/src/templates/SharedResearchOutput.tsx

```diff
@@ -52,72 +80,153 @@ const SharedResearchOutput: React.FC<SharedResearchOutputProps> = ({
+              <div css={{ paddingBottom: `${12 / perRem}em` }}>
+                <Headline2 styleAsHeading={4}>Description</Headline2>
+                <RichText poorText text={description} />
+              </div>
+            )}
+            {description && !!tags.length && <Divider />}
+            {!!tags.length && (
+              <>
+                <Headline2 styleAsHeading={4}>Tags</Headline2>
+                <TagList tags={tags} />
+              </>
+            )}
+          </Card>
+        )}
+        {accessInstructions && (
+          <Card>
             <div css={{ paddingBottom: `${12 / perRem}em` }}>
-              <Headline2 styleAsHeading={4}>Description</Headline2>
-              <RichText poorText text={description} />
+              <Headline2 styleAsHeading={4}>Access Instructions</Headline2>
+              <RichText poorText text={accessInstructions} />
             </div>
-          )}
-          {description && !!tags.length && <Divider />}
-          {!!tags.length && (
-            <>
-              <Headline2 styleAsHeading={4}>Tags</Headline2>
-              <TagList tags={tags} />
-            </>
-          )}
-        </Card>
-      )}
-      {accessInstructions && (
+          </Card>
+        )}
         <Card>
-          <div css={{ paddingBottom: `${12 / perRem}em` }}>
-            <Headline2 styleAsHeading={4}>Access Instructions</Headline2>
-            <RichText poorText text={accessInstructions} />
-          </div>
+          <Headline2 styleAsHeading={4}>Additional Information</Headline2>
+          <ol css={additionalInformationListStyles}>
+            <li css={additionalInformationEntryStyles}>
+              <strong>Sharing Status</strong>
+              <span css={additionalInformationValueStyles}>
+                {sharingStatus}
+              </span>
+            </li>
+            {asapFunded === undefined || (
+              <>
+                <Divider />
+                <li css={additionalInformationEntryStyles}>
+                  <strong>ASAP Funded</strong>
+                  <span css={additionalInformationValueStyles}>
+                    {asapFunded ? 'Yes' : 'No'}
+                  </span>
+                </li>
+              </>
+            )}
+            {usedInPublication === undefined || (
+              <>
+                <Divider />
+                <li css={additionalInformationEntryStyles}>
+                  <strong>Used in a Publication</strong>
+                  <span css={additionalInformationValueStyles}>
+                    {usedInPublication ? 'Yes' : 'No'}
+                  </span>
+                </li>
+              </>
+            )}
+            {doi === undefined || (
+              <>
+                <Divider />
+                <li css={additionalInformationEntryStyles}>
+                  <strong>Identifier (DOI)</strong>
+                  <span css={additionalInformationValueStyles}>
+                    <Anchor href={`https://doi.org/${doi}`}>
```

Comment:
Could you please construct this url using new URL().toString(). We do this everywhere to ensure a link is always valid